### ISSUE TITLE
Crash to Title Screen on Tag Creation

### ISSUE THREAD

**Issue by FaustHeinrich** [Doridian/factorio-fox-todo#16]
When placing a Tag with "TODO: ", I get the following error message.

Mod Version: Fox TODO 2.0.0
Game Version: 2.0.8 (build 79416, win64)

```
Error while running event fox-todo::on_chart_tag_added (ID 154)
__fox-todo__/gui.lua:16: attempt to concatenate local 'rt_type' (a nil value)
stack traceback:
	__fox-todo__/gui.lua:16: in function 'signal_id_to_rich_text'
	__fox-todo__/gui.lua:20: in function 'get_tag_caption'
	__fox-todo__/gui.lua:190: in function 'render_todo_gui_player'
	__fox-todo__/gui.lua:211: in function 'render_todo_gui_force'
	__fox-todo__/tags.lua:24: in function 'on_tag_added'
	__fox-todo__/tags.lua:112: in function <__fox-todo__/tags.lua:111>
```

Thought, I'd let you know... Thank you for your mod!

**Comment by Doridian** [comment]
Tiny API change with 2.0 I missed. Sorry about that, fixed now in 2.0.1!

**Comment by FaustHeinrich** [comment]
That fast fast. Thank you 👍🥇
